cos-fleet-catalog-tools/camel-connector-maven-plugin/src/main/java/org/bf2/cos/catalog/camel/maven/connector/GenerateAppMojo.java

```diff
@@ -99,81 +101,85 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             this.indexFile = indexPath.toPath().resolve("connectors.json");
             this.manifestLocalFile = definitionPathLocal.toPath().resolve(this.manifestId + ".json");
 
-            if (!Files.exists(manifestLocalFile)) {
+            if (!Files.exists(manifestLocalFile) && !exclude) {
                 getLog().warn("Skipping App build as the definition file " + manifestLocalFile.getFileName() + " is missing");
```

Comment:
message is now misleading

Reply:
This is to retain the old behavior and message when the connector is **not** marked for exclusion.

Added a new log message to log the exclusion: https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/pull/569/files#diff-2645cfc5dc6af74108a4a5c8ae14480e5818d63285a0d605ce2cd2bccf32226cR182